components/ActionSheet/src/private/MDCActionSheetHeaderView.m

```diff
@@ -0,0 +1,258 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCActionSheetHeaderView.h"
+
+#import "MaterialTypography.h"
+
+static const CGFloat kTitleLabelAlpha = 0.87f;
+static const CGFloat kMessageLabelAlpha = 0.6f;
+static const CGFloat kMessageOnlyPadding = 23.f;
+static const CGFloat kLeadingPadding = 16.f;
+static const CGFloat kStandardPadding = 16.f;
+static const CGFloat kTrailingPadding = 16.f;
+static const CGFloat kTitleOnlyPadding = 18.f;
+static const CGFloat kMiddlePadding = 8.f;
+
+@interface MDCActionSheetHeaderView ()
+
+@property(nonatomic, nonnull, strong) UIScrollView *scrollView;
+
+@end
+
+@implementation MDCActionSheetHeaderView {
+  UILabel *titleLabel;
+  UILabel *messageLabel;
+}
+
+@synthesize mdc_adjustsFontForContentSizeCategory;
```

Comment:
You have a custom getter/setter for this so you don't want to synthesize.
You should define BOOL _mdc_adjustsFontForContentSizeCategory; in you @implementation just above.

Reply:
Same as in MDCActionSheetController, using `@synthesize` and no extra ivar in the `@implemenation` declaration.